web/frontend/src/components/slam-2d-render.vue

```diff
@@ -161,16 +332,29 @@ const updateCloud = (pointcloud: Uint8Array) => {
     }
   }
 
-  scene.add(points);
-  scene.add(marker);
-  scene.add(intersectionPlane);
-};
+  // construct grid spaced at 1 meter
+  const gridHelper = createGridHelper(points);
+
+  // construct axes
+  const axesPos = createAxisHelper('AxesPos', Math.PI / 2);
+  axesPos.rotateY(Math.PI / 2);
```

Comment:
[opt] add `axesPos.rotateX(0);` for consistency. Completely optional

Reply:
I will skip this addition since it does nothing and could serve as a point of confusion.